apps/openmw/mwbase/world.hpp

```diff
@@ -269,6 +269,8 @@ namespace MWBase
             virtual MWWorld::Ptr  getFacedObject() = 0;
             ///< Return pointer to the object the player is looking at, if it is within activation range
 
+            virtual float getMaxActivationDistance() = 0;
```

Comment:
The implementation for this function doesn't seem to be included in this pull request. Forgot to add it?